tc_pdf/handlers/pdf.py

```diff
@@ -51,10 +50,9 @@ def regex(cls, has_unsafe_or_hash=True):
 
         return ''.join(reg)
 
-    @gen.coroutine
-    def get(self, **kwargs):
+    async def get(self, **kwargs):
         # check if request is valid
-        yield gen.maybe_future(self.check_pdf(kwargs.copy()))
+        await self.check_pdf(kwargs.copy())
```

Comment:
@intellisense Same here, this is indeed a problem, but it's not introduced by the changes that we made to provide Thumbor 7 / Python 3 compatibility, is it? Are yo sure you want to fix this in this PR or maybe open another?

Reply:
Agreed, however there was a change that I forgot to comment now requested [here](https://github.com/intellisense/tc_pdf/pull/3#discussion_r1195226983). If you are comfortable making this change lets do it otherwise I can do it separately. Thanks!